apps/ui/src/stores/spaces.ts

```diff
@@ -107,6 +111,8 @@ export const useSpacesStore = defineStore('spaces', () => {
     fetch,
     fetchMore,
     fetchSpace,
-    toggleSpaceStar
+    toggleSpaceStar,
+    protocol,
```

Comment:
nit. not sure if it makes sense, but we return and import those grouped by type (first refs, then functions), so protocol and handleProtocolChange shouldn't be next to each other.